portfolio_style.css

```diff
@@ -51,39 +46,50 @@ h1, h2 {
 	
 }
 
-.portfolio {
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	padding-left: 50px;
-	padding-right: 50px;
-	
-}
-
 .welcome {	
 	background-color: #f7daed;
 	font-size: 30pt;
 	padding-bottom: 200px;
 	height: 90%;
 }
 
+.contact {
+	background-color: #f7daed;
+	font-size: 20pt;
+	padding-bottom: 200px;
+	padding-top: 100px;
+	height: 90%;
+	text-align: center;
+}
+
 .links {
 	font-size:20pt;
 	text-align: center;
 }
 
-/*.portfolio_page {
+.portfolio_page {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	padding-left: 50px;
+	padding-right: 50px;
 	background-color: #f7daed;
 	font-size: 10pt;
 	padding-bottom: 200px;
 	height: 90%;
-*/
+}
+
 article, footer {
 	padding-left: 150px;
 	padding-right: 150px;
 
 }
 
+.picture-of-me {
+	height: 220px;
+	
+}
+
 footer {
 	text-align: center;
```

Comment:
You might want to change your sizes to percentages to make sure they will work with the mobile site, also I am not sure if the no-wrap element will work with mobiles without seeing the end product. You might also want to remove the bits that you have commented out if you are not using them. Overall looks great though!

Reply:
Hi, thanks for your comments. Good points!
I will work on the percentages next I think

Reply:
Hi Louse. I did the percentage bit and erased my comments I am not using. Thanks. When I update it you might have to go over them again (I will need your approval)